src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,78 @@
-public class StringManipulation implements StringManipulationInterface {
+public class StringManipulation {
```

Comment:
This class is supposed to implement StringManipulationInterface according to the assignment skeleton. I do see that you changed your test class to allow for StringManipulation to exist separately from StringManipulationInterface, but you may want to change it back as Sara mentioned that she will be running everyone's code against her own tests which may require StringManipulation to implement StringManipulationInterface.